Library/Formula/fontconfig.rb

```diff
@@ -1,7 +1,8 @@
 class Fontconfig < Formula
   desc "XML-based font configuration API for X Windows"
   homepage "http://fontconfig.org/"
-  url "http://fontconfig.org/release/fontconfig-2.11.1.tar.bz2"
+  url "https://www.freedesktop.org/software/fontconfig/release/fontconfig-2.11.1.tar.bz2"
+  mirror "http://fontconfig.org/release/fontconfig-2.11.1.tar.bz2"
```

Comment:
It looks like the freedesktop URL may actually be the legit one now. The homepage automatically redirects to https://www.freedesktop.org/wiki/. Should perhaps set the homepage to `https://wiki.freedesktop.org/www/Software/fontconfig/` and drop the mirror.